apps/src/gamelab/animationLibrary.json

```diff
@@ -5184,7 +5184,7 @@
     "ore_iron": [
       "ore_iron"
     ],
-    "ore_ironAlt": [
+    "ore_ironalt": [
```

Comment:
There's no consistency between camelCase, underscore_names, and names with spaces. Are these all taken from the Kenny file names? It's weird they are so inconsistent.